sunpy/map/sources/gong.py

```diff
@@ -61,3 +71,56 @@ def spatial_units(self):
     @property
     def observer_coordinate(self):
         return get_earth(self.date)
+
+
+class GONGHalphaMap(GenericMap):
+    """
+    GONG H-Alpha Map.
+
+    The Global Oscillation Network Group (GONG) operates a six-station network of H-alpha
+    imagers located around the Earth that observe the Sun nearly continuously.
+
+    References
+    ----------
+    * `GONG H-Alpha Page <https://nso.edu/data/nisp-data/h-alpha/>`_
+    * `GONG H-Alpha Observation Details <https://nispdata.nso.edu/webProdDesc2/presenter.php?file=halpha_fulldisk_images_overview.html&echoExact=0&name=Overview%20:%20GONG%20H-alpha%20Full-disk%20Images>`_
+    * `GONG Header Keywords <https://gong.nso.edu/data/HEADER_KEY.html>`_
+    * `DOI:/10.25668/as28-7p13 <https://doi.org/10.25668/as28-7p13>`_
+    """
+    def __init__(self, data, header, **kwargs):
+        header['WCSNAME'] = 'Helioprojective-Cartesian'
+        header['CTYPE1'] = 'HPLN-TAN'
+        header['CTYPE2'] = 'HPLT-TAN'
+        super().__init__(data, header, **kwargs)
+
+    @classmethod
+    def is_datasource_for(cls, data, header, **kwargs):
+        return (str(header.get('TELESCOP', '')).endswith('GONG') and
+                str(header.get('IMTYPE', '')).startswith('H-ALPHA'))
+
+    @property
+    def date(self):
+        return Time(self.meta.get('date-obs'))
+
+    @property
+    def scale(self):
+        return SpatialPair((self.meta['SOLAR-R'] * u.arcsec) / (self.meta['FNDLMBMI'] * u.pixel),
+                           (self.meta['SOLAR-R'] * u.arcsec) / (self.meta['FNDLMBMA'] * u.pixel))
```

Comment:
```suggestion
        solar_r = self.meta['SOLAR-R'] * u.arcsec
        return SpatialPair(solar_r / (self.meta['FNDLMBMI'] * u.pixel),
                           solar_r / (self.meta['FNDLMBMA'] * u.pixel))
```

Should we have some way to bail out if these keys don't exist? I don't know if that's a reasonable case to consider.